app/components/Edit/EditSections.js

```diff
@@ -13,6 +13,136 @@ import * as dataService from '../../utils/DataService';
 import { getAuthRequestHeaders } from '../../utils/index';
 import { daysOfTheWeek } from '../../utils/index';
 
+function getDiffObject(curr, orig) {
```

Comment:
I'm a little confused by what's going on here. Why are there all the function definitions here duplicates of methods lower in the file? I also vaguely feel like I was the one who originally wrote these, so did we accidentally revert my changes at some point? I don't really care which way we go with, but we definitely shouldn't have two sets of the same functions twice in this file.

Reply:
We must have both been working on this file at the same time. I had a merge conflict and wasn't 100% sure what you were working on. Was this the only function you changed in this file? @richardxia

Reply:
Looking back at https://github.com/ShelterTechSF/askdarcel-web/pull/265/files, it looks like I totally refactored the file in order to get it to pass the linter, so I ended up hoisting all the methods that didn't actually use `this` to be standalone functions. I don't think it's worth trying to get the linter to pass again right now, but we definitely shouldn't have duplicate function definitions. It looks like most of the standalone functions that you added were duplicates of methods on `EditSections`, so we should collapse them and just stick to one set.